switchdev_utils/config-service/switchdev_network_config

```diff
@@ -566,15 +570,23 @@ def validate_config(config):
             logger.error(err)
             raise ValueError(err)
         try:
-            if int(sriov_pf.get("numOfVfs")):
-                continue
+            int(sriov_pf.get("numOfVfs"))
         except ValueError as value_err:
-            logger.error("numOfVfs for %s must be number" %
-                         sriov_pf.get("name"))
+            err = "numOfVfs for %s must be number" % sriov_pf.get("name")
```

Comment:
why do you need this assignment ? and not just logger.error("numOfVfs....")

Reply:
done